.github/ISSUE_TEMPLATE/task.yml

```diff
@@ -1,14 +1,14 @@
-name: Enhancement
-description: Suggest an enhancement or improvement for the project
-title: "[ENHANCEMENT] "
-labels: "enhancement"
+name: Task
+description: Suggest an improvement to the current feature
```

Comment:
I think the description can be more like, `Dev cleanup, maintainence, or administrative tasks`

Reply:
@sunsplat changes were made!